pkg/common/gatewayapi_utils.go

```diff
@@ -17,7 +17,9 @@ import (
 	gatewayapiv1alpha2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
 )
 
-const GatewayProgrammedConditionType = "Programmed"
+const (
+	GatewayProgrammedConditionType = "Programmed"
+)
```

Comment:
Linter? I actually prefer the one-line version. Am I the only the one?

Reply:
Just a matter of personal taste. 

I would rather go for grouping several constant declarations, even if it's only one. If another constant needs to be added, the git diff is cleaner.